web/Services/TenantService.cs

```diff
@@ -64,12 +64,14 @@ private IQueryable<TenantDbModel> GetFilteredTenantQuery(HousingRequirementModel
         if (!isMatching)
         {
             return tenantQuery.Where(t => (filters.AcceptsPets == null || t.HasPet == filters.AcceptsPets)
-                                          && (filters.AcceptsNotInEET == null || t.NotInEET == filters.AcceptsNotInEET)
+                                          && (filters.AcceptsNotInEET == null || t.InEET == !filters.AcceptsNotInEET)
                                           && (filters.AcceptsCredit == null
                                               || t.UniversalCredit == filters.AcceptsCredit)
                                           && (filters.AcceptsBenefits == null
                                               || t.HousingBenefits == filters.AcceptsBenefits)
-                                          && (filters.AcceptsUnder35 == null || t.Under35 == filters.AcceptsUnder35));
+                                          && (filters.AcceptsUnder35 == null || t.Under35 == filters.AcceptsUnder35)
+                                          && (filters.AcceptsWithoutGuarantor == null
+                                              || t.HasGuarantor == filters.AcceptsWithoutGuarantor));

```

Comment:
I think that should be `t.HasGuarantor == !filters.AcceptsWithoutGuarantor`, shouldn't it?
e.g. if `HasGuarantor` is false, the tenant doesn't have a guarantor, so we'd want `AcceptsWithoutGuarantor` to be true